boards/nxp/frdm_imx93/frdm_imx93_mimx9352_a55.dts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_mimx93_a55.dtsi>
+#include "frdm_imx93-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "NXP i.MX93 A55";
+	compatible = "fsl,mimx93";
+
+	chosen {
+		zephyr,console = &lpuart2;
+		zephyr,shell-uart = &lpuart2;
+		/* sram node actually locates at DDR DRAM */
```

Comment:
do we still need this comment? we've been using the DDR for a while now...

Reply:
please drop. This comment adds no new information. It is already evident that we're not using SRAM from the node name and its address space.